stellargraph/mapper/graph_generator.py

```diff
@@ -96,6 +96,16 @@ def flow(self, graph_ilocs, targets=None, batch_size=1, name=None):
                     f"expected targets to be the same length as node_ids, found {len(targets)} vs {len(graph_ilocs)}"
                 )
 
+        if not isinstance(batch_size, int):
+            raise TypeError(
+                f"expected batch_size to be integer type, found {type(batch_size)}"
```

Comment:
I often forget this too, but `type(...)` returns a class value that doesn't format very nicely, and one instead has to pull out the (string) name manually:

```suggestion
                f"expected batch_size to be integer type, found {type(batch_size).__name__}"
```

```python
>>> batch_size = 1.2
>>> print(f"expected batch_size to be integer type, found {type(batch_size)}")
expected batch_size to be integer type, found <class 'float'>
>>> print(f"expected batch_size to be integer type, found {type(batch_size).__name__}")
expected batch_size to be integer type, found float
```